test/unit/query-generator/create-schema-query.test.ts

```diff
@@ -0,0 +1,36 @@
+import { buildInvalidOptionReceivedError } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/check.js';
+import { expectsql, getTestDialect, sequelize } from '../../support';
+
+const dialectName = getTestDialect();
+
+const notSupportedError = new Error(`Schemas are not supported in ${dialectName}.`);
+
+describe('QueryGenerator#createSchemaQuery', () => {
```

Comment:
We had an unit test for both `collate` and `chatset` before, but it's not here. Is that something that we want to add or do we feel it's redundant?

Reply:
Done